### ISSUE TITLE
cpu is at high usage but it is not updating on nanopool that any shares have been mined

### ISSUE THREAD

**Issue by wpask2** [UnamSanctam/SilentXMRMiner#518]
first 5 minutes of testing and got around 5 shares, and so then i leave it running for almost two hours now and it shows that it is inactive when on my test pc aka my rdp it shows the high usage under file explorer, almost as if its mining, but on nanopool it shows that the worker is inactive. help?

**Comment by UnamSanctam** [comment]
Can you run the checker and post what it returns? If it's using CPU then it's mining (since otherwise it can't use the CPU) so the shares should appear on the pool. Sometimes pools are slow to update their statistics but eventually things should appear.

**Comment by wpask2** [comment]
where is the checker at?

**Comment by UnamSanctam** [comment]
Oh, you should use my new miner: https://github.com/UnamSanctam/SilentCryptoMiner, when you compile that a checker will get complied together with the miner that you can use to test/check.

**Comment by wpask2** [comment]
ok so i used the new one and i turned off stealth but still it does not show that explorer.exe is using up alot of usage and my cpu is at like less than 10% now and doesnt appear to be mining anything, but when i run the checker it says watchdog running: yes
and under miners it says nothing

**Comment by UnamSanctam** [comment]
Then it's not running, post your settings here. You can press "Save" on the "Main" tab and then zip it and post that file here which is the easiest or you can screenshot the builder.

You might have to change the "Startup" tabs "Filename" to something other than Microsoft since some anticheats block that folder.

**Comment by wpask2** [comment]
ok i did as you said and changed the name to 64os.exe and when i run the checker it says it works now

**Comment by wpask2** [comment]
but my question is, if i upload the checker file alone to mega and download it on my real pc and run it, will it still check for me?

**Comment by UnamSanctam** [comment]
Yes will check for the miner (that you built) on any computer you send it to.

**Comment by wpask2** [comment]
alright

**Comment by wpask2** [comment]
one last question.... i notice your name is unam sanctam

**Comment by wpask2** [comment]
are you of south korean heritage by chance

**Comment by UnamSanctam** [comment]
Haha no I'm Swedish.

**Comment by wpask2** [comment]
ah ok lol

**Comment by wpask2** [comment]
also when i ended the nslookup in task manager which is what i assume the miner used to cloak itself, after i ended it, it did not start up again

**Comment by UnamSanctam** [comment]
The program it injects into depends on what you choose, but the watchdog will start it up again after about 3 minutes of being closed so that it won't be too fast.

**Comment by wpask2** [comment]
another question is, the uninstall and checker and the miner starter all disappeared from the directory i had it in and was changed to a prefetch file for some reason, how do i uninstall it without the uninstaller?

**Comment by UnamSanctam** [comment]
You can uninstall it manually https://github.com/UnamSanctam/SilentXMRMiner/wiki#how-do-i-remove-the-miner.

**Comment by wpask2** [comment]
ok also there is another issue, i used your unam downloader to bind a program setup and the my xmr stub made with your updated crypto miner, however it does not seem to work. when i run the stub by itself and check it with the checker it says it is running, but when i run the binded file it does not work.

**Comment by UnamSanctam** [comment]
Is the download link you entered into the downloader correct? Does it work if you use the Unam Binder (less can go wrong with the binder)?

**Comment by wpask2** [comment]
i have not tried unam binder yet, but let me do that.

**Comment by wpask2** [comment]
i have updated the link, but it still says "this program may of not have been installed correctly" when running the binded file. let me see if the checker works though

**Comment by wpask2** [comment]
Ok its picking it up now

**Comment by UnamSanctam** [comment]
Ok because if it says "this program may of not have been installed correctly" then you've forgotten .exe after the filename when binding it.

**Comment by wpask2** [comment]
![image](https://user-images.githubusercontent.com/100374414/156873470-a56bdde2-d196-49a8-82d6-963353d96e07.png)
do have .exe on both

**Comment by wpask2** [comment]
ok so i did some testing and it does get flagged by windows defender. even when added to windows defender exclusions. anyway to fix?

**Comment by UnamSanctam** [comment]
Are you running it as administrator? It can't add exclusions without that (which is what the shield means).

**Comment by wpask2** [comment]
yes so i have updated it and checked "run as administrator" and it still flags it. also when i download it onto my rdp it automatically flags it aswell.

**Comment by UnamSanctam** [comment]
Do you mean that the bound file (that the binder generated) gets detected or that some file (like the miner that gets extracted) gets detected after running the bound file?

**Comment by wpask2** [comment]
the bound file

**Comment by wpask2** [comment]
i compiled it and uploaded it and when i downloaded it, it gets flagged with this
![image](https://user-images.githubusercontent.com/100374414/156967949-96dd5474-0175-4fa4-bb4b-a5551b761292.png)

**Comment by wpask2** [comment]
even when compiled with a start delay and run as admin and add to windows defender exclusions

**Comment by UnamSanctam** [comment]
Yes of course it will be detected, it's a free and open source project on GitHub, meaning that it will get detected. Every project on GitHub that can be used maliciously will always become detected sooner or later.

To circumvent detections you'd probably need a crypter.

**Comment by wpask2** [comment]
do you have any suggestions for a good crypter?

**Comment by UnamSanctam** [comment]
No I haven't used (or purchased) any crypters in a few years so all those that I know don't really exist anymore (and my experiences with those aren't that relevant anymore even if they still existed).

**Comment by wpask2** [comment]
hey so whenever i try making a gpu silent miner it never works. ive tried rvn, etc, and eth all with no avail. any idea what may cause issues? ive put in all the correct values and everything but it never starts up when i run it.

**Comment by UnamSanctam** [comment]
What GPU do you have? Can you run the checker and post what it returns?

**Comment by wpask2** [comment]
i have an nvidia gtx 1060 max q design. its a laptop graphics card. when i run the checker it shows "compatible gpus" and shows my gpu 0 and gpu 1, 0 being the integrated graphics and 1 being my 1060. and under miners it is blank. i noticed it does not even start injecting the process and get my computer at a high gpu usage

**Comment by UnamSanctam** [comment]
Can you try the example settings at the top of the wiki https://github.com/UnamSanctam/SilentCryptoMiner/wiki? In case it's because of your settings.

**Comment by wpask2** [comment]
i believe i have everything done correctly, should i send you the json output?
-

**Comment by UnamSanctam** [comment]
Sure.

**Comment by wpask2** [comment]
also, with the wallet address input, do i have to do the whole format of   WalletAddress.WorkerName/WithdrawalPasswordor can i just paste my wallet address?

**Comment by UnamSanctam** [comment]
For nanopool you would enter the whole format just so that you can edit the minimum payout, but if you don't want to change that or have a worker name then just entering the wallet address works fine.

**Comment by wpask2** [comment]
Hi so it seems that when I use your new miner and create a stub for the rvn miner, and then bind it with either unam binder or downloader, it does not work

**Comment by UnamSanctam** [comment]
What does it say if you run the checker?
